src/modules/airline/index.ts

```diff
@@ -0,0 +1,237 @@
+   */
+  readonly name: string;
+  /**
+   * The IATA code of the airport (e.g. `'DFW'`).
+   */
+  readonly iataCode: string;
+}
+
+const numerics = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+const visuallySimilarCharacters = ['0', 'O', '1', 'I', 'L'];
+const aircraftTypeMaxRows = {
+  regional: 20,
+  narrowbody: 35,
+  widebody: 60,
+};
+const aircraftTypeSeats = {
+  regional: ['A', 'B', 'C', 'D'],
+  narrowbody: ['A', 'B', 'C', 'D', 'E', 'F'],
+  widebody: ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'J', 'K'],
+};
+
+/**
+ * Module to generate airline related data.
+ */
+export class AirlineModule {
+  constructor(private readonly faker: Faker) {
+    // Bind `this` so namespaced is working correctly
+    for (const name of Object.getOwnPropertyNames(AirlineModule.prototype)) {
+      if (name === 'constructor' || typeof this[name] !== 'function') {
+        continue;
+      }
+
+      this[name] = this[name].bind(this);
+    }
+  }
+
+  /**
+   * Generates a random airport.
+   *
+   * @example
+   * faker.airline.airport() // { name: 'Dallas Fort Worth International Airport', iataCode: 'DFW' }
+   *
+   * @since 8.0.0
+   */
+  airport(): Airport {
+    return this.faker.helpers.arrayElement(
+      this.faker.definitions.airline.airport
+    );
+  }
+
+  /**
+   * Generates a random airline.
+   *
+   * @example
+   * faker.airline.airline() // { name: 'American Airlines', iataCode: 'AA' }
+   *
+   * @since 8.0.0
+   */
+  airline(): Airline {
+    return this.faker.helpers.arrayElement(
+      this.faker.definitions.airline.airline
+    );
+  }
+
+  /**
+   * Generates a random airplane.
+   *
+   * @example
+   * faker.airline.airplane() // { name: 'Airbus A321neo', iataTypeCode: '32Q' }
+   *
+   * @since 8.0.0
+   */
+  airplane(): Airplane {
+    return this.faker.helpers.arrayElement(
+      this.faker.definitions.airline.airplane
+    );
+  }
+
+  /**
+   * Generates a random [record locator](https://en.wikipedia.org/wiki/Record_locator). Record locators
+   * are used by airlines to identify reservations. They're also known as booking reference numbers,
+   * locator codes, confirmation codes, or reservation codes.
+   *
+   * @param options The options to use. Defaults to `{}`.
+   * @param options.allowNumerics Whether to allow numeric characters. Defaults to `false`.
+   * @param options.allowVisuallySimilarCharacters Whether to allow visually similar characters such as '1' and 'I'. Defaults to `false`.
+   *
+   * @example
+   * faker.airline.recordLocator() // 'KIFRWE'
+   * faker.airline.recordLocator({ allowNumerics: true }) // 'E5TYEM'
+   * faker.airline.recordLocator({ allowVisuallySimilarCharacters: true }) // 'ANZNEI'
+   * faker.airline.recordLocator({ allowNumerics: true, allowVisuallySimilarCharacters: true }) // '1Z2Z3E'
+   *
+   * @since 8.0.0
+   */
+  recordLocator(
+    options: {
+      allowNumerics?: boolean;
+      allowVisuallySimilarCharacters?: boolean;
```

Comment:
Please update your branch and then add JSDocs to these option params like in other modules

Here it would be something like

```suggestion
      /**
       * Whether to allow numeric characters.
       *
       * @default false
       */
      allowNumerics?: boolean;
      /**
       * Whether to allow visually similar characters such as '1' and 'I'.
       *
       * @default false
       */
      allowVisuallySimilarCharacters?: boolean;
```

Reply:
Done! Thanks for the suggestions.